sparktestingbase/testcase.py

```diff
@@ -84,11 +121,11 @@ def tearDownClass(cls):
         Tear down the basic panda spark test case. This stops the running
         context and does a hack to prevent Akka rebinding on the same port.
         """
-        cls.sc.stop()
-        derby_path = os.path.join(os.getcwd(), 'metastore_db')
+        print("stopping class")
```

Comment:
Do we need to leave this debug logging in?

Reply:
removed